app/modules/net.c

```diff
@@ -1539,6 +1738,14 @@ static const LUA_REG_TYPE net_array_map[] = {
 };
 #endif
 
+static const LUA_REG_TYPE net_cert_map[] = {
+  { LSTRKEY( "verify" ), 	LFUNCVAL( net_cert_verify ) },  
+#ifdef CLIENT_SSL_CERT_AUTH_ENABLE
+  { LSTRKEY( "auth" ),		LFUNCVAL( net_cert_auth ) }, 
```

Comment:
You are correct. I have a version of the documentation that does include it, but this command doesn't actually work -- the espressif code throws a null pointer exception. I'm trying to get information out of espressif about whether this functionality ever worked. I felt that once I can make it work, I'll add the documentation section. The default for the define is to not include this code.